how-to-be-a-developer/readme.md

```diff
@@ -0,0 +1,25 @@
+> Goal of this guide: learn about what the role of a developer is at New Atoms
+
+## It's all about the perspective
+
+Being a developer is a way of looking at certain situations.
+For example: if a client wants to add something to a produced article, the conclusions drawn from this request can be very different depending on the perspective:
+* Was there anything wrong with the content?
+* How, if applicable, can the company structurally integrate this feedback into the production process?
+* What effect will ignoring the request on the relationship with the costumer?
+* How can the experience of giving feedback be optimized?
```

Comment:
So I guess these are all different perspectives that can belong to a certain developer role? maybe put the role behind the perspectives? or the other way around? Not yet clear enough for me.